tests/unit/runtime/test_ds_config_dict.py

```diff
@@ -21,7 +21,7 @@ class TestBasicConfig(DistributedTest):
     world_size = 1
 
     def test_cuda(self):
```

Comment:
Maybe rename `test_cuda` to `test_accelerator`?

Reply:
Will rename to `test_accelerator`